relations/tests/test_artist_relations.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
This test requires a musicbrainz-docker setup, so I've removed it from the travis setup for now.